docs/updates/2019-04.md

```diff
@@ -0,0 +1,145 @@
+application. One possible explanation to this behavior is that these visitors are curious about the tool, but lacking
+a running engine or a configured Qlik Sense Enterprise prevented them from further exploring. To mitigate this, a
+[demo app](https://catwalk.core.qlik.com/?engine_url=wss://apps.core.qlik.com/app/doc/01775889-c700-413f-9b0e-6ba1837c52b0/)
+is automatically loaded and can be used as an introduction to the tool.
+
+### Default websocket
+
+If you are mainly working with one engine instance, the new default socket connection feature might be of use.
+By clicking on the star, top right in catwalk, the current websocket will be stored in the browser's local storage and
+used in future catwalk sessions.
+
+![screenshot](../images/catwalk-default-websocket.png)
+
+### Add support for selections in "always one selected value" fields
+
+A field with the [property](https://core.qlik.com/services/qix-engine/apis/qix/definitions/#nxfieldproperties)
+`qOneAndOnlyOne` will enforce only one selected value. The selection model in catwalk has been updated accordingly.
+
+![screenshot](../images/catwalk-field-select-one-value.gif)
+
+### Visual support for loosely coupled tables
+
+We have added support for loosely couple tables (tables with a circular reference). Normally this is not intended in
+a data model and catwalk will help out by marking the involved fields with a dashed red line.
+
+![screenshot](../images/catwalk-circular-reference.png)
+
+### Export hypercube data and definition
+
+In one of the feature requests there was an expressed need for exporting hypercube data. It sounded like an
+interesting feature so we decided to build support for it and also offer a way to export the hypercube definition to
+clipboard. Both features can be found in the hypercube toolbar.
+
+![screenshot](../images/catwalk-export-hypercube.png)
+
+### Save open hypercubes
+
+Something which has bugged us for a while is that all hypercubes created in catwalk gets lost when the window is
+refreshed or closed. To prevent this we now store the hypercube definition for each app in the browser's local storage.
+
+### Keep table headers during scrolling
+
+To avoid losing track of table headers when scrolling large apps, the table names are now kept on top. When the table
+name disappear, there is no more fields in the table. When having complex applications with a lot of associations, it's
+common to have large gaps between the fields and it might be easy to miss the fields in the bottom.
+
+![screenshot](../images/catwalk-scrolling-headers.gif)
+
+### Various improvements
+
+Apart from the larger fixes we have made a couple of smaller improvements:
+
+* add support for table and field comments
+* handle rendering issues when layout isn't retrieved and while loading the layout
+* Multiple table-names properly displayed in tool tips
+* Fix key field sorting during selection in other fields
+
+## corectl 1.0.0 beta release
```

Comment:
Can we link to the download here? Either `https://github.com/qlik-oss/corectl#download` or `https://github.com/qlik-oss/corectl/releases/tag/v1.0.0-beta.1`